H2MLauncher.UI/MainWindow.xaml

```diff
@@ -16,7 +16,8 @@
         KeyDown="Window_KeyDown"
         Icon="Assets/icon.ico"
         xmlns:i="http://schemas.microsoft.com/xaml/behaviors"
-        Title="Better H2M-Launcher" Height="450" Width="800">
+        Title="Better H2M-Launcher"  Height="576" Width="1024">
```

Comment:
We like to keep the UI neat, so try to maximize the space without making it a lot bigger. 
As when I ran the update I saw there was a lot more space. 
![image](https://github.com/user-attachments/assets/d5580f87-b9f6-4fd2-8f37-c96acd3fc22d)

Reply:
![image](https://github.com/user-attachments/assets/cc2d9fad-eb6b-4312-a21a-c8f2b7140acf)


Better now ? It narrowed

Reply:
Move the text for Total players more to the left so it is in line with the rest.
Also, either make the filter bar bigger or add the text `Filter:` in front of it to fill up the empty space.

Reply:
![image](https://github.com/user-attachments/assets/1ea820a4-8ed3-477b-84c0-da0528bd91dc)

What do you think now

Reply:
What happened to the numbers after `Total Server`? 
Try to make it such that the line after `Ping` is the end. So no more space after that. 
You can do that by increasing the width of the `Hostname` column so it moves the other columns farther away.

Reply:
![image](https://github.com/user-attachments/assets/63f7db25-d334-4407-990a-da46e0b26ea3)


Did it :)